src/user.js

```diff
@@ -33,12 +48,30 @@ class User {
     });
   }
 
+  addToCook(recipe) {
+    if (typeof recipe === 'object') {
+      this.recipesToCook.push(recipe)
+    }
+  }
+
+  removeFromRecipesToCook(recipe) {
+    let cooked = this.recipesToCook.indexOf(recipe);
+    this.recipesToCook.splice(cooked, 1)
+  }
+
+  filterRecipesToCook(tag) {
+    return this.recipesToCook.filter(recipe => {
+      return recipe.tags.includes(tag);
+    })
+  }
+
+
   static getUserData(userId) {
     const userUrl = 'https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData'
     const promise = fetch(userUrl)
       .then(response => response.json())
       .then(data => data.wcUsersData.find(user => user.id === userId))
-    return promise; 
+    return promise;
```

Comment:
This is something we talked before but just want to leave it here so we don't forget and that is implementing error handling.